src/main/java/by/gexateq/simplerestservice/service/impl/ReviewServiceImpl.java

```diff
@@ -49,23 +47,27 @@ public void checkAndUpdateReviews() {
     @Override
     public void reviewsProcessing() {
 
-        // a slow logic....
         LocalDateTime currentDate = LocalDateTime.now();
         LocalDateTime threeDaysAgo = currentDate.minusDays(3);
         var reviews = reviewRepository.
                 findByCreatedAtBeforeAndStatusNot(threeDaysAgo, ReviewStatus.CANCELLED);
         log.debug("reviewsProcessing: Found {} reviews", reviews.size());
+
         reviews.forEach(r -> {
             if (r.getCreatedAt().isBefore(LocalDateTime.now().minusDays(1))) {
-                log.info("Review id={} is a day before", r.getId());
-                try {
-                    Thread.sleep(1000);
-                } catch (InterruptedException ie) {
-                    log.error("InterruptedException: ", ie);
-                }
+                executorService.submit(() -> {
```

Comment:
Let it be as initial logic so I approve it for now. We will discuss and do additional update.